Perf.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off
```

Comment:
Can you try rebasing on the latest upstream main? I think this will disappear once you do that.